packages/api/src/modules/transaction/controller.ts

```diff
@@ -576,6 +579,7 @@ export class TransactionController {
           signer,
           predicateId: predicateId ?? undefined,
           type,
+          id,
```

Comment:
if you recive this id, just return an list with:

result: [await transactionService.findById()]